src/features/staking/StakingModal.tsx

```diff
@@ -125,7 +125,7 @@ export const StakingModal = () => {
     insufficientBalance: isInsufficientBalance,
   });
 
-  const isAmountInvalid = Number(stakingAmountFormatted) * 10 ** 4 <= 0;
+  const isAmountInvalid = Number(stakingAmountFormatted || 0) * 10 ** 4 <= 0;
```

Comment:
`10000n`?

Reply:
Updated - unnecessary multiplication anyway if it's only checking for >0